.github/workflows/ci.yml

```diff
@@ -14,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+        uses: actions/checkout@v3
```

Comment:
Could you use the git SHA of the latest version of the checkout action? This is a security feature. See https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas

Reply:
I've added the SHA now.